client/src/components/exercise/lab10/components/code/TrainingAICodeBlock.js

```diff
@@ -0,0 +1,355 @@
+        actions.setTrainingDuration(value);
+      }
+    }
+  };
+
+  return (
+    <div className="code_editor">
+      <div className="code_editor__content">
+        <div className="code_editor__files">
+          {/* Simulation.js */}
+          <div className="code_editor__file code_editor__file--active">
+            Simulation.js
+          </div>
+        </div>
+
+        <div className="code_editor__code">
+          <div className="code_editor__line">
+            <span className="code_editor__line--purple">import&nbsp;</span>
+            <span className="code_editor__line--blue">React</span>
+            <span className="code_editor__line--gold">&nbsp;</span>
+            <span className="code_editor__line--purple">from&nbsp;</span>
+            <span className="code_editor__line--orange">
+              &lsquo;react&lsquo;
+            </span>
+            <span className="code_editor__line--gold">;</span>
+          </div>
+
+          <div className="code_editor__line">
+            <span className="code_editor__line--purple">import&nbsp;</span>
+            <span className="code_editor__line--gold">&#123;</span>
+            <span className="code_editor__line--blue">
+              {" convertMinutesToSeconds"}
+            </span>
+            <span>{", "}</span>
+            <span className="code_editor__line--blue">{"handleShiftLeft"}</span>
+            <span>{", "}</span>
+            <span className="code_editor__line--blue">
+              {"handleShiftRight"}
+            </span>
+            <span>{", "}</span>
+            <span className="code_editor__line--blue">
+              {"useCollectUserData "}
+            </span>
+            <span className="code_editor__line--gold">&#125;&nbsp;</span>
+            <span className="code_editor__line--purple">from&nbsp;</span>
+            <span className="code_editor__line--orange">
+              &lsquo;../utils&lsquo;
+            </span>
+            <span className="code_editor__line--gold">;</span>
+          </div>
+
+          <div className="code_editor__line">
+            <span className="code_editor__line--purple">import&nbsp;</span>
+            <span className="code_editor__line--gold">&#123;</span>
+            <span className="code_editor__line--blue">
+              {" collectSimulationData "}
+            </span>
+            <span className="code_editor__line--gold">&#125;&nbsp;</span>
+            <span className="code_editor__line--purple">from&nbsp;</span>
+            <span className="code_editor__line--orange">
+              &lsquo;../data/NeuralNetwork&lsquo;
+            </span>
+            <span className="code_editor__line--gold">;</span>
+          </div>
+
+          <div className="code_editor__line">&nbsp;</div>
+
+          <div className="code_editor__line">
+            <span className="code_editor__line--purple">const</span>
+            <span className="code_editor__line--yellow">
+              {" SIMULATION_DURATION_SECONDS "}
```

Comment:
I think it could possibly be useful to add a comment in the codeblock to indicate that this is the value that needs to be changed, like the previous repair section.
![Screenshot 2023-11-13 at 2 35 24 PM](https://github.com/all-rit/ALL/assets/54957497/c74ad42d-2990-4a3b-a677-5d63a2d4fcab)

Reply:
will do